runtime/src/main/java/org/corfudb/runtime/RebootUtil.java

```diff
@@ -0,0 +1,131 @@
+package org.corfudb.runtime;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.CorfuRuntime.CorfuRuntimeParameters;
+import org.corfudb.runtime.clients.*;
+import org.corfudb.runtime.view.Layout;
+import org.corfudb.util.CFUtils;
+import org.corfudb.util.NodeLocator;
+import org.corfudb.util.Sleep;
+
+import java.time.Duration;
+
+/**
+ * Utility to Reboot a server which includes reset or restart
+ * Reset wipes out all existing Corfu data while restart does not.
+ *
+ * <p>Created by WenbinZhu on 4/18/19.
+ */
+@Slf4j
+public class RebootUtil {
```

Comment:
If only unit tests use RebootUtil, why it seats in runtime?

Reply:
This is for user to do reset/restart